### PR TITLE
de-verbositying the admin log for grenades

### PR DESCRIPTION
**Pull request by Tk420634**
because BOY am I tired of seeing that, lol

## Pre-Merge Checklist
- [ ] You tested this on a local server.
- [ ] This code did not runtime during testing.
- [ ] You documented all of your changes.

## Changelog
:cl:
add: Added new things
add: Added more things
del: Removed old things
tweak: tweaked a few things
balance: rebalanced something
fix: fixed a few things
soundadd: added a new sound thingy
sounddel: removed an old sound thingy
imageadd: added some icons and images
imagedel: deleted some icons and images
spellcheck: fixed a few typos
code: changed some code
refactor: refactored some code
config: changed some config setting
admin: messed with admin stuff
server: something server ops should know
/:cl:

### CODE REVIEW

File: code/modules/research/designs/ammolathe_designs.dm
Comment (Tk420634) [inline code/modules/research/designs/ammolathe_designs.dm:1]: lol

File: code/modules/hydroponics/grown/misc.dm [hunk 1]
```diff
@@ -232,8 +232,8 @@
 
 /obj/item/reagent_containers/food/snacks/grown/cherry_bomb/attack_self(mob/living/user)
 	user.visible_message(span_warning("[user] plucks the stem from [src]!"), span_userdanger("You pluck the stem from [src], which begins to hiss loudly!"))
-	message_admins("[ADMIN_LOOKUPFLW(user)] primed a cherry bomb for detonation at [ADMIN_VERBOSEJMP(user)]")
-	log_game("[key_name(user)] primed a cherry bomb for detonation at [AREACOORD(user)].")
+	message_admins("[ADMIN_LOOKUPFLW(user)] primed a cherry bomb  at [ADMIN_VERBOSEJMP(user)]")
```
Comment (DameonOwen) [inline code/modules/hydroponics/grown/misc.dm:235]: extra space before "at"

**Review by DameonOwen — CHANGES_REQUESTED** [review]

File: code/modules/research/designs/ammolathe_designs.dm
Comment (TalkingCactus) [reply code/modules/research/designs/ammolathe_designs.dm:1]: lol

**Review by TalkingCactus — APPROVED** [review]

File: code/modules/hydroponics/grown/misc.dm (hunk 1 above)
Comment (Tk420634) [reply code/modules/hydroponics/grown/misc.dm:235]: *augh

**Review by DameonOwen — APPROVED** [review]

**Pull request merged**